dspace-xmlui/src/main/java/org/dspace/app/xmlui/aspect/submission/submit/DescribeStep.java

```diff
@@ -64,238 +69,224 @@
-    protected static final Message T_report_no=
-        message("xmlui.Submission.submit.DescribeStep.report_no");
-        
-    /**
-     * A shared resource of the inputs reader. The 'inputs' are the
-     * questions we ask the user to describe an item during the
-     * submission process. The reader is a utility class to read
-     * that configuration file.
-     */
-    private static DCInputsReader INPUTS_READER = null;
+            message("xmlui.Submission.submit.DescribeStep.head");
+
+    protected static final Message T_unknown_field =
+            message("xmlui.Submission.submit.DescribeStep.unknown_field");
+
+    protected static final Message T_required_field =
+            message("xmlui.Submission.submit.DescribeStep.required_field");
+
+    protected static final Message T_last_name_help =
+            message("xmlui.Submission.submit.DescribeStep.last_name_help");
+
+    protected static final Message T_first_name_help =
+            message("xmlui.Submission.submit.DescribeStep.first_name_help");
+
+    protected static final Message T_year =
+            message("xmlui.Submission.submit.DescribeStep.year");
+
+    protected static final Message T_month =
+            message("xmlui.Submission.submit.DescribeStep.month");
+
+    protected static final Message T_day =
+            message("xmlui.Submission.submit.DescribeStep.day");
+
+    protected static final Message T_series_name =
+            message("xmlui.Submission.submit.DescribeStep.series_name");
+
+    protected static final Message T_report_no =
+            message("xmlui.Submission.submit.DescribeStep.report_no");
 
     private static final Message T_vocabulary_link = message("xmlui.Submission.submit.DescribeStep.controlledvocabulary.link");
 
     protected ItemService itemService = ContentServiceFactory.getInstance().getItemService();
+
     protected ChoiceAuthorityService choiceAuthorityService = ContentAuthorityServiceFactory.getInstance().getChoiceAuthorityService();
+
     protected MetadataAuthorityService metadataAuthorityService = ContentAuthorityServiceFactory.getInstance().getMetadataAuthorityService();
 
+    protected DCInputsReader inputsReader = null;
+
     /**
-     * Ensure that the inputs reader has been initialized, this method may be
-     * called multiple times with no ill-effect.
+     * Establish our required parameters.  AbstractStep will enforce these.
      */
-    private static void initializeInputsReader() throws DCInputsReaderException
+    public DescribeStep()
+    {
+        this.requireSubmission = true;
+        this.requireStep = true;
+    }
+
+    @Override
+    public void setup(SourceResolver resolver, Map objectModel, String src, Parameters parameters) throws ProcessingException, SAXException, IOException
     {
-        if (INPUTS_READER == null)
+        super.setup(resolver, objectModel, src, parameters);
+
+        String formFileName = I18nUtil.getInputFormsFileName(context.getCurrentLocale());
```

Comment:
I think this is the only necessary change to fix DS-3598. So I'm not sure why we need all these other changes.